spec/datadog/tracing/contrib/rails/utils_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'lib/datadog/tracing/contrib/rails/utils'
+require 'rails/version'
+
+RSpec.describe Datadog::Tracing::Contrib::Rails::Utils do
+  describe 'railtie_supported?' do
+    subject(:railtie_supported?) { described_class.railtie_supported? }
+
+    before do
+      stub_const('::Rails::Railtie', instance_double('::Rails::Railtie'))
+    end
```

Comment:
This constant isn't defined on Rails 2, so it should be moved to the Rails 3 block